docs/actions-scripts.md

```diff
@@ -1,23 +1,103 @@
 The most common kind of action is a scripted action.
 
 Generally-speaking, you can write whatever code you like as long as it will run successfully on server, and it is possible to [test this locally](actions-pipelines.md#running-your-code-locally).
+
 However, note the following restrictions and guidance:
 
 * **Write analyses in Python, R, or Stata.**
 You can can use more than one language in a single project if necessary.  You can find more information about the available libraries below.
+
 * **Do not write code that requires an internet connection to run.**
 Any research objects (datasets, libraries, etc.) that are retrieved via the internet should be imported to the repo locally first.
 If this is not possible (for instance if the object size is too large to be transferred via GitHub) then get in touch.
+
 * **Avoid code that consumes a lot of time or memory.** The server is not an infinite resource. We can advise on code optimisation if run-times become problematic.  A good strategy is to split your processing into separate project pipeline actions; the job runner can then choose to run them in parallel if sufficient resources are available.
-* **Write code that runs across different platforms.**
-Since code will be run both locally and within a Linux-based Docker environment. In practice, this just means ensuring you use forward-slashes `/` for directories.
+
+* **Write code that runs in the OpenSAFELY platform.**
+Code will be run within a Linux-based Docker environment. In practice, this just means ensuring you use forward-slashes `/` for directories.
+
 * **Structure your code into discrete chunks, both within scripts, and by splitting into different pipeline actions.**
 This helps with:
 	* readability
 	* bug-finding
 	* parallelisation via the project pipeline
 
 
+## Reading and Writing Outputs
+
+Scripted actions can read and write output files that are saved in the workspace. These generally fall into two categories: large files of `highly_sensitive` data to feed into other actions, or smaller `moderately_sensitive` outputs intended for review and release.
```

Comment:
suggestion:

I tend to favour using lists in technical writing where possible. Feel free to decide whether you prefer this or ignore it.

```suggestion
Scripted actions can read and write output files that are saved in the workspace. These generally fall into two categories:
* large files of `highly_sensitive` data for use by other actions
* smaller `moderately_sensitive` outputs for review and release
```